support-frontend/assets/pages/contributions-landing/components/PaymentMethodSelector.jsx

```diff
@@ -220,6 +224,7 @@ function withProps(props: PropTypes) {
           }
           {contributionTypeIsRecurring(props.contributionType) &&
           fullExistingPaymentMethods.map((existingPaymentMethod: RecentlySignedInExistingPaymentMethod) => (
+            // TODO: check is this gets error state.
```

Comment:
Is this TODO an accidental commit, or are you planning on testing it in CODE?

Reply:
Yeah we're gonna test this in CODE now!